src/NuGet.Core/NuGet.Protocol.Core.v3/ReportAbuseResourceV3.cs

```diff
@@ -32,50 +33,29 @@ public ReportAbuseResourceV3()
         /// <returns>The first URL from the resource, with the URI template applied.</returns>
         public Uri GetReportAbuseUrl(string id, NuGetVersion version)
         {
-            //return Utility.ApplyPackageIdVersionToUriTemplate(_reportAbuseTemplates.First(), id, version);
-            return new Uri(String.Format(CultureInfo.InvariantCulture, "https://www.nuget.org/packages/{0}/{1}/ReportAbuse", id, version.ToNormalizedString()));
+            try
+            {
+                var uriString = _uriTemplate
+                   .Replace("{id}", id)
+                   .Replace("{version}", version.ToNormalizedString());
+
+                return new Uri(uriString);
+            }
+            catch (UriFormatException)
```

Comment:
Agreed. It's a left-behind piece of code that can be removed. Addressed with https://github.com/NuGet/NuGet.Client/commit/98b8d996c94ba9ef8664f23fb95ce450d0779b45